Reqnroll/BoDi/ObjectContainer.cs

```diff
@@ -500,7 +500,7 @@ public bool IsRegistered(Type type, string name = null)
     private void AssertNotResolved(RegistrationKey interfaceType)
     {
         if (_resolvedKeys.Contains(interfaceType))
-            throw new ObjectContainerException("An object has been resolved for this interface already.", null);
+            throw new ObjectContainerException($"An object has been resolved for this interface ({interfaceType.ToString()}) already.", null);
```

Comment:
The grammar here is a little off. Here's an alternative suggestion:

```suggestion
            throw new ObjectContainerException($"An object has been resolved for the interface \"{interfaceType.ToString()}\" already.", null);
```